### PR TITLE
add options to SwiftLayoutPrinter

### PR DESCRIPTION
**Pull request by oozoofrog**
- add options

### CODE REVIEW

File: Sources/SwiftLayout/Printing/SwiftLayoutPrinter.swift [hunk 1]
```diff
@@ -8,6 +8,22 @@
...
+        public init(rawValue: Int) {
+            self.rawValue = rawValue
+        }
+        
+        /// print system constraints
+        public static let withSystemConstraints: PrintOptions = .init(rawValue: 1)
+        /// print view only have accessibility identifier
+        public static let onlyIdentifier: PrintOptions = .init(rawValue: 1 << 1)
+        /// print with view config
+        public static let withViewConfig: PrintOptions = .init(rawValue: 1 << 2)
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Printing/SwiftLayoutPrinter.swift:24]: Is this option currently available?
It seems that this option is not yet available

ps. In fact, it is a very necessary option when preparing for performance tests. 
I'm very surprised that the options is included in this pr.

**Review by gmlwhdtjd — APPROVED** [review]
I'm very happy with it, except for the part where the `withViewConfig` option doesn't work yet.

File: Sources/SwiftLayout/Printing/SwiftLayoutPrinter.swift (hunk 1 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Printing/SwiftLayoutPrinter.swift:24]: hahaha it works(a little bit) in next PR.

**Comment by oozoofrog** [comment]
> I'm very happy with it, except for the part where the `withViewConfig` option doesn't work yet.

you can see soon. (may be not perfect), I wish features be forced from you. @gmlwhdtjd

**Pull request merged**